index.js

```diff
@@ -222,8 +228,8 @@ module.exports = postcss.plugin( 'postcss-flexboxfixer', function( opts ) {
     }
 
     return function( css ) {
-        css.walkDecls( /(^background$)/, function( decl ) {
-            if(!/-webkit(-\w*|)-gradient/.test(decl.value)){
+        css.walkDecls(function(decl) {
+            if (!/-webkit-gradient/.test(decl.value)) {
```

Comment:
@gucong3000 Yes, I get that :) I'm asking about changing 
```
/-webkit(-\w*|)-gradient/
```
to
```
/-webkit-gradient/
```
The former is meant to handle CSS using any of ``-webkit-gradient``, ``-webkit-linear-gradient``, or ``-webkit-radial-gradient``. Did you decide to remove support for those two older forms, or was it an accident?